### PR TITLE
Added installation of shared libraries

### LINKED ISSUE #57: Strange cmake problem on MSVC, how to link OGDF correctly?
I'm trying to go my first steps with a small test that links against OGDF. But I fail miserably in the build step on MSVC. On Linux and macOS everything works like a charm! Here is the relevant excerpt from my cmake configuration:

```
find_package(Ogdf REQUIRED)
target_link_libraries(${TESTNAME} PRIVATE OGDF GTest::GTest)
```

Is that the correct way to link OGDF? I tried this on Linux and macOS and it works. The cmake configuration also works on MSVC. And I tested in cmake that the `OGDF` target exists with `if(NOT TARGET OGDF) ... fail in error`.
But when I try to build with `ninja` on MSVC, I get the following error:
```
ninja: error: 'OGDF-NOTFOUND', needed by 'OGDFTest_autogen/timestamp', missing and no known rule to make it
```

Does that mean anything to anyone? Is my CMakeLists.txt wrong? I'm completely stuck in this step and have basically no idea what may be causing it...

I also found a cmake config file called `ogdf-config.cmake` that claims one should include the file when using `find_package()` (see https://github.com/ogdf/ogdf/blob/master/cmake/ogdf-config.cmake) but I failed to understand how that works...

### PR DESCRIPTION
**Pull request by emmenlau**
This PR adds a minor change to the build instructions. It allows to install a shared build of OGDF into the `bin` directory. Without this PR, shared libraries are currently not installed.

Closes #57

### CODE REVIEW

File: cmake/coin.cmake [hunk 1]
```diff
@@ -56,13 +56,15 @@ endif()
 
 # installation
 set(COIN_INSTALL_LIBRARY_DIR "lib/${CMAKE_LIBRARY_ARCHITECTURE}" CACHE PATH "Installation path of COIN library")
+set(OGDF_INSTALL_BIN_DIR "bin" CACHE PATH "Installation path of OGDF library runtime")
```
Comment (milsen) [inline cmake/coin.cmake:59]:
```suggestion
set(COIN_INSTALL_BIN_DIR "bin" CACHE PATH "Installation path of COIN runtime targets")
```

File: cmake/coin.cmake [hunk 2]
```diff
@@ -56,13 +56,15 @@ endif()
...
 set(COIN_INSTALL_INCLUDE_DIR "include" CACHE PATH "Installation path of COIN header files (creates subdirectory)")
 set(COIN_INSTALL_CMAKE_DIR "lib/${CMAKE_LIBRARY_ARCHITECTURE}/cmake/OGDF/" CACHE PATH "Installation path of COIN files for CMake")
 mark_as_advanced(COIN_INSTALL_LIBRARY_DIR COIN_INSTALL_INCLUDE_DIR COIN_INSTALL_CMAKE_DIR)
```
Comment (milsen) [inline cmake/coin.cmake:62]:
```suggestion
mark_as_advanced(COIN_INSTALL_LIBRARY_DIR COIN_INSTALL_BIN_DIR COIN_INSTALL_INCLUDE_DIR COIN_INSTALL_CMAKE_DIR)
```

File: cmake/coin.cmake [hunk 3]
```diff
@@ -56,13 +56,15 @@ endif()
...
 install(TARGETS COIN
   EXPORT CoinTargets
   LIBRARY DESTINATION "${COIN_INSTALL_LIBRARY_DIR}"
   ARCHIVE DESTINATION "${COIN_INSTALL_LIBRARY_DIR}"
+  RUNTIME DESTINATION "${OGDF_INSTALL_BIN_DIR}"
```
Comment (milsen) [inline cmake/coin.cmake:67]:
```suggestion
  RUNTIME DESTINATION "${COIN_INSTALL_BIN_DIR}"
```

File: cmake/ogdf.cmake [hunk 4]
```diff
@@ -168,6 +168,7 @@ endif()
 
 # installation
 set(OGDF_INSTALL_LIBRARY_DIR "lib/${CMAKE_LIBRARY_ARCHITECTURE}" CACHE PATH "Installation path of OGDF library")
+set(OGDF_INSTALL_BIN_DIR "bin" CACHE PATH "Installation path of OGDF library runtime")
```
Comment (milsen) [inline cmake/ogdf.cmake:171]:
```suggestion
set(OGDF_INSTALL_BIN_DIR "bin" CACHE PATH "Installation path of OGDF runtime targets")
```

File: cmake/ogdf.cmake [hunk 5]
```diff
@@ -168,6 +168,7 @@ endif()
...
 set(OGDF_INSTALL_INCLUDE_DIR "include" CACHE PATH "Installation path of OGDF header files (creates subdirectory)")
 set(OGDF_INSTALL_CMAKE_DIR "lib/${CMAKE_LIBRARY_ARCHITECTURE}/cmake/OGDF/" CACHE PATH "Installation path of OGDF files for CMake")
 mark_as_advanced(OGDF_INSTALL_LIBRARY_DIR OGDF_INSTALL_INCLUDE_DIR OGDF_INSTALL_CMAKE_DIR)
```
Comment (milsen) [inline cmake/ogdf.cmake:174]:
```suggestion
mark_as_advanced(OGDF_INSTALL_LIBRARY_DIR OGDF_INSTALL_BIN_DIR OGDF_INSTALL_INCLUDE_DIR OGDF_INSTALL_CMAKE_DIR)
```

**Review by milsen — CHANGES_REQUESTED** [review]
Thank you for your figuring this out and fixing it!
It would be nice if you could add my requested changes to your commit; other than that this looks good.

**Comment by emmenlau** [comment]
Dear @milsen thanks for the thumbs up!

I've implemented your requested changes and squashed the commits (this is usually requested, but you did not explicitly say so, so I hope that was your expectation?).

Thanks for considering this PR, please review and/or merge!

**Comment by milsen** [comment]
Thank you (also for squashing the commits). This PR looks good now, I'll merge it soon.

**Review by milsen — APPROVED** [review]

**Pull request merged**